images/hook/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/dumb-init /bin/sh
```

Comment:
It's just unnecessary mixing of concerns IMHO. And I think in this specific case the actual resulting process tree is: `dumb-init` -> `dumb-init` -> `sh`.